scripts/qemu-check.sh

```diff
@@ -139,7 +139,7 @@ do
 			SHM_MBOX=qemu-bridge-hp-sram-mem
 			has_rom=true
 			;;
-		imx8 | imx8x | imx8m)
+		imx8 | imx8x | imx8m | imx8ulp)
 			READY_IPC="00 00 00 00 00 00 04 c0"
```

Comment:
this is for qemu support. I don't think we have qemu support for hifi4 integration in imx8ulp. So, I think we can skip it for now